cmd/gendoc/main.go

```diff
@@ -256,7 +256,7 @@ func getModuleNameFromGoMod(goModPath string) (string, error) {
 //
 // The import path is based on the path of source files in the package and the module name in the nearest go.mod file.
 // Exits the program with an error return code in case of an error.
-func mustGetPackageName(astRoot string, f *ast.Package) string {
+func mustGetPackageName(astRoot string, f *ast.Package) string { //nolint: staticcheck
```

Comment:
I am going to create a minor task for this one since the the `ast.Package` is deprecated.